#hipnuc config
imu_serial: "/dev/ttyUSB0"
baud_rate: 115200
frame_id: "base_link"
imu_topic: "/IMU_data"
parent_frame: "odom"  # 固定框架，可以是 "odom" 或 "map"

#hipnuc data package 
frame_id_costom: "base_link_hipnuc"
imu_topic_costom: "/imu_package_hipnuc"

